client/src/components/Review/AllReviewsPanel.js

```diff
@@ -1,95 +1,72 @@
+// IMPORTS
 import React from "react";
-import Popup from "reactjs-popup";
 import styled from "styled-components";
 import axios from 'axios';
 
+// STYLED COMPONENTS
+const StyleModal = styled.div`
+  display: flex;
+  flex-direction: column;
 
-const Button = styled.button`
-  background: ${props => props.primary ? "palevioletred" : "white"};
-  color: ${props => props.primary ? "white" : "palevioletred"};
+  padding: 10px;
 
-  font-size: 1em;
-  margin: 1em;
-  padding: 0.25em 1em;
-  border: 2px solid palevioletred;
-  border-radius: 3px;
+  font-size: 12px;
 `
 
-const StyleModal = styled.div`
-    font-size: 12px;
-  `
-
-  const Header = styled.div`
-    width: 100%;
-    border-bottom: 1px solid gray;
-    font-size: 18px;
-    text-align: center;
-    padding: 5px;
- `
-  const Content = styled.div`
+const Header = styled.div`
+  text-align: center;
+  font-size: 20px;
+  font-weight: bold;
   
-    width: 100%;
-    padding: 10px 5px;
-  `
-  const Actions = styled.div`
-    width: 100%;
-    padding: 10px 5px;
-    margin: auto;
-    text-align: center;
-  `
+  color: #FBD702;
+
+  width: 100%;
+  margin-bottom: 15px;
+`
 
+const Content = styled.div`
+  display: flex;
 
-    class AllReviewsPanel extends React.Component {
-      state = {
-        reviews: []
-      }
-    
-      componentDidMount() {
-        axios.get(`https://wheretocode-master.herokuapp.com/reviews/`)
-          .then(res => {
-            const reviews = res.data;
-            this.setState({ reviews });
-          })
-      }
-    
-      render() {
-        return (
-          <StyleModal>
-    <Header> Reviews </Header>
-    <Content>      
-    <ul>
+  padding: 15px;
+  margin: 5px 0 5px 0;
+
+  border-radius: 10px 10px 10px 10px;
+  background-color: white;
+`
+// COMPONENT
+class AllReviewsPanel extends React.Component {
+  // STATE
+  state = {
+    reviews: []
+  }
+
+  // METHODS
+  componentDidMount() {
+    axios.get(`https://wheretocode-master.herokuapp.com/reviews/`)
+      .then(res => {
+        const reviews = res.data;
+        this.setState({ reviews });
+      })
+  }
+
+  // RENDER
+  render() {
+    return (
+      <StyleModal>
+        <Header> Reviews </Header>
+        <Content>      
+          <ul className='ratingInfo'>
             { this.state.reviews.map(review =>
-            <li>UserId: {review.user_id},  Rating: {review.rating}, Comments:{review.comments}</li>)}
+              <li>
```

Comment:
This li tag will require a key or this can cause rendering issues down the line. Best to take care of this early to avoid weird bugs.